### PR TITLE
integration nlpcloud handler

### LINKED ISSUE #8024: [Integration] Integrate NLPCloud with MindsDB
### Is there an existing integration?

- [X] I have searched the existing integrations.

### Short description and motivation for the proposed feature

The goal of this issue is to build a new handler to integrate NLPCloud with MindsDB.

### Resources 💡

🏁 To get started, please refer to the following resources:

📚 [Building Data Handler docs](https://docs.mindsdb.com/contribute/data-handlers): This documentation will guide you through the process of building a custom handler for MindsDB. It contains essential information on the handler structure, methods, and best practices.

📖 [NLPCloud](https://nlpcloud.com/)

📖 [NLPCloud API](https://docs.nlpcloud.com/?python#endpoints)

👉 [Twitter Handler](https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/twitter_handler): This existing handler connects MindsDB with the Twitter API, allowing users to automate tasks related to posting and analyzing tweets. You can use the Twitter Handler as a reference for building the Slack API integration.

👉 [Binance API Handler](https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/binance_handler): Another example of a successful MindsDB integration to the Binance API to make trade data available to use for model training and predictions.

### Next Steps 🕵️‍♂️ 🕵️‍♀️

You can familiarize yourself with NLPCloud.
Create a new handler that connects MindsDB to NLPCloud.
Implement the core features mentioned above.
Thoroughly test the integration to ensure proper functionality.
Submit a pull request with the new NLPCloud Integration handler.

### Implementation

_No response_

### 📢 Additional rewards 🏆

This integration brings 7️⃣ 0️⃣ : points for SWAG and entry into the draw for a 👇 :

💻 [Razer Blade 16 Laptop](https://www.razer.com/gaming-laptops/Razer-Blade-16/RZ09-0483SEJ3-R3U1)
👕 🐻 [MindsDB Swag](https://store.covver.io/mindsdb/collections/swag-collection)
ℹ️ For more info check out https://mindsdb.com/hacktoberfest/ 👈

### PR DESCRIPTION
**Pull request by Better-Boy**
## Description

NLPCloud handler

Fixes #8024 

## Type of change

(Please delete options that are not relevant)

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [x] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [x] 📄 This change requires a documentation update

## Verification Process

To ensure the changes are working as expected:

 - [ ]   Test Location: Specify the URL or path for testing.
 - [ ]   Verification Steps: Outline the steps or queries needed to validate the change. Include any data, configurations, or actions required to reproduce or see the new functionality.

## Additional Media:

- [x] I have attached a brief loom video or screenshots showcasing the new functionality or change.

Screenshot 2023-10-24 at 6 02 05 AM

Screenshot 2023-10-24 at 6 09 17 AM

## Checklist:

- [x] My code follows the style guidelines(PEP 8) of MindsDB.
- [x] I have appropriately commented on my code, especially in complex areas.
- [x] Necessary documentation updates are either made or tracked in issues.
- [x] Relevant unit and integration tests are updated or added.

### CODE REVIEW

**Review by paxcema — CHANGES_REQUESTED** [review]
Maybe I'm missing something, but why wouldn't this integration be an ML engine instead of an API handler? I.e. inherit `BaseMLEngine` instead.

@ZoranPandovski any thoughts?

File: mindsdb/integrations/handlers/nlpcloud_handler/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+## Implemented Features
+
+- [x] NLPCloud
+  - [x] Support SELECT
+    - [x] Support LIMIT
+    - [x] Support WHERE
+    - [x] Support ORDER BY
+    - [x] Support column selection
+
+## TODO
```
[the remark this answers is not in the document]
Comment (Better-Boy) [reply mindsdb/integrations/handlers/nlpcloud_handler/README.md]: Done

**Comment by chandrevdw31** [comment]
Hey @Better-Boy if your credits are exhausted and you are unable to convert this to ML handler for now, you can convert the PR to a draft so long and wait until you have the credits to work on this contribution

**Comment by paxcema** [comment]
Indeed, the route @chandrevdw31 is the right option. As it stands, I'm afraid we cannot merge this as an API handler.

**Comment by Better-Boy** [comment]
Thanks @chandrevdw31 @paxcema . I'll convert this PR to draft.

**Comment by Better-Boy** [comment]
Closing this PR due to securing another API key is not possible at the moment and nlpcloud hasn't responded to my mails either.

**Pull request closed without merging**